python/lsst/verify/specset.py

```diff
@@ -687,11 +689,11 @@ def update(self, other):
         """Merge another `SpecificationSet` into this one.
 
         Parameters
-        ----------
+        ---------
         other : `SpecificationSet`
-            Another `SpecificationSet`. `Specification`\ s in ``other`` that do
-            not exist in this set are added to this one. `Specification`\ s in
-            ``other`` replace specifications of the same name in this one.
+            Another `SpecificationSet`. Specification in ``other`` that do
+            exist in this set are added to this one. Specification in ``other``
```

Comment:
A side note -- it would be nice to have the formatting changes in a separate commit.  No need to rebase this to do that, but for future reference.

Reply:
Oh shoot. That's a rebase error. Thanks for catching these.